modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/css/components/ShareModalContent.scss

```diff
@@ -6,6 +6,16 @@ html#{$cadmin-selector} {
 			max-height: 368px;
 		}
 
+		.list-group-item {
+			.autofit-col.dropdown-options {
+				padding: 0;
+			}
+
+			.autofit-col:first-of-type {
```

Comment:
is there any option to remove this custom style, by using an utility class?

Reply:
Sounds good, I'll use `pl-0` instead!